theforeman/foreman
# Fixes #37167 - Subscription entitlement report template

1. Remove subscription entitlement report
2. Update Subscription - general report to also show subscription expiration dates including expiry filter.

To test:

1. Run db:migrate and db:seed
2. Ensure that the Subscription - Entitlement Report is deleted.
3. Run the Subscription - General Report and ensure you see subscription expiration filter before generating report and field on the generated report.
